File: concorrencia.domain/Posto.cs
using System;
namespace concorrencia.domain
{
    public class Posto
    {
        public int id { get; set; }
        public string NomePosto { get; set; }
        public string Tipo { get; set; }
        public Bandeira Bandeira { get; set; }
        public int BandeiraId { get; set; }
        public Regiao Regiao { get; set; }
        public int RegiaoId { get; set; }
    }
}

File: concorrencia-app/src/app/_models/Posto.ts
import { Regiao } from './Regiao';
import { Bandeira } from './Bandeira';

export class Posto {
  id: number;
  regiaoId: number;
  bandeiraId: number;
  nomePosto: string;
  tipo: string;
  bandeira: Bandeira;
  regiao: Regiao;
}
